### PR TITLE
add github action to run dennis lint on every commit

### PR DESCRIPTION
**Pull request by LeoMcA**
After #30 I realized it would be incredibly trivial to get our linter running on every commit.

@akatsoulas r?

@peiying2 as written this will just add a status to each commit (shown as a green tick, or a red x), as can be seen on this deliberate error I introduced into my fork: https://github.com/LeoMcA/sumo-l10n/commit/88cd3a12b779e99e3ebc7d0b0fe6954e3a0ad76a

Clicking on the red x, details, will show the actual error: https://github.com/LeoMcA/sumo-l10n/runs/854900186

We could make this noisier if you wanted and trigger a slack (or, I guess, matrix) message.

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Lint Strings
+
+on: push
```
Comment (akatsoulas) [inline .github/workflows/lint.yml:3]: let's contain this to the master branch

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+jobs:
+  main:
```
Comment (akatsoulas) [inline .github/workflows/lint.yml:6]: s/main/lint or something like that

File: .github/workflows/lint.yml (hunk 1 above)
Comment (peiying2) [reply .github/workflows/lint.yml]: I missed this request. If you need a technical review from l10n, I would involve someone else from my team. I don't think I will add any value to this, :). Let me know if you do.

**Comment by peiying2** [comment]
@LeoMcA Thanks for the ping. For mozilla.org (Fluent), Pontoon can alert contributor when one attempts to submit a translation. For others, this might not be the case. It will be detected during sync between Pontoon and the repo (FxA). In FxA, I will be notified and I usually go to Pontoon to fix it. If it involves knowledge of the language, then I will comment in the string, and the contributor will be notified. I think I can be the first line of defense for this, unless we can trigger the warning in Pontoon. I don't know how often an error occurs. Let's catch them before the day to merge with the master. 

In short, what you have demonstrated with your example is excellent. It shows me the file name, the language, and the string itself. Let's go with this solution.

**Comment by akatsoulas** [comment]
@LeoMcA let me know if this is ready to be merged

**Comment by akatsoulas** [comment]
@peiying2 can we give write access to @LeoMcA for this repo?

**Comment by peiying2** [comment]
> @peiying2 can we give write access to @LeoMcA for this repo?

If this is ready to merge, I will merge it now. I will find out access issue.

**Comment by LeoMcA** [comment]
Yep, ready to merge

**Comment by LeoMcA** [comment]
As discussed, we'll do something similar to this, but instead running from a non-l10n repo.
